src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private final Map<String, Integer> storage = new HashMap<>();

```

Comment:
The `storage` map should not be `final` since it's contents are intended to be modified.